### PR TITLE
Make clear that you need to reload storage nodes

### PR DESCRIPTION
**Pull request by mkpankov**
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1586)

### CODE REVIEW

File: vagrant/README.md [hunk 1]
```diff
@@ -58,13 +58,17 @@ From here you can decide what type of setup to run.
 - Monitored Ldiskfs:
 
   ```sh
-  vagrant provision --provision-with=install-ldiskfs-no-iml,configure-lustre-network,create-ldiskfs-fs,create-ldiskfs-fs2,mount-ldiskfs-fs,mount-ldiskfs-fs2
+  vagrant provision --provision-with=install-ldiskfs-no-iml
+  vagrant reload mds1 mds2 oss1 oss2
```
Comment (ip1981) [inline vagrant/README.md]: I  think it needs a comment on why.

**Comment by mkpankov** [comment]
@jgrund sure, but it's much more significant investment. This change is already tested and slightly improves the current situation, and we can then improve significantly using the reboot provisioning.

File: vagrant/README.md (hunk 1 above)
Comment (mkpankov) [reply vagrant/README.md]: Done

**Comment by jgrund** [comment]
> @jgrund sure, but it's much more significant investment. This change is already tested and slightly improves the current situation, and we can then improve significantly using the reboot provisioning.

reboot provisioning has landed, we should be able to use the full provisioning steps now.

**Comment by mkpankov** [comment]
Closing in favor https://github.com/whamcloud/integrated-manager-for-lustre/pull/1701

**Pull request closed without merging**